### ISSUE TITLE
history chat no works

### ISSUE THREAD

**Issue by josep112** [RocketChat/Rocket.Chat#21347]
### Description:

I can not look in the history of the message, it gets blinking and hangs the screen

### Steps to reproduce:

In a message in Livechat Customer Click on the icon on the right side above: Historic of chat contact, choose one of the messages that are going to appear, click on top after this is blinking 3 icons and nothing happens

### Expected behavior:

Which appears the selected conversation

### Actual behavior:

The screen is waiting and nothing happens

### Server Setup Information:

Version of Rocket.Chat Server: 3.5.1
Operating System: Debian 9
Deployment Method: tar
Number of Running Instances: 1
NodeJS Version: v12.18.2
MongoDB Version: v4.0.19

### Client Setup Information

Desktop App or Browser Version: 89.0.4389.82 (Versão oficial) (x86_64)
Operating System: MacOs Mojane 10.14.6

### Additional context
<img width="1280" alt="Captura de Tela 2021-03-29 às 14 34 17" src="https://user-images.githubusercontent.com/12038013/112876505-fa622a00-909b-11eb-97a0-4f51e5ba8c90.png">

### Relevant logs:

I20210329-12:50:25.301(-3) API ➔ debug GET: /api/v1/channels.messages/?roomId=kevp8kYN38kqfktjf&count=50&offset=0&sort={%22ts%22:%201}&query={%22$or%22:%20[%20{%22t%22:%20{%22$exists%22:%20false}%20},%20{%22t%22:%20%22livechat-close%22}%20]%20}

**Comment by dudanogueira** [comment]
HI! I had this problem.

the user role that are trying to use this feature **cannot have the permission _view-joined-room_**
and also, **it must has the permission _view-c-room_**

please, check those permission at the Admin > Permissions. For some reason, those permissions changed somehow here for me and was having this very same issue.

this is the part of the code that manage this API:
https://github.com/RocketChat/Rocket.Chat/blob/2ad2d0bd7ed7aa03b7eb1f23bee29f0af62a0fe7/app/api/server/v1/channels.js#L572

**Comment by josep112** [comment]
Hi, thank you very much, the view-joined-room was unchecked but the problem still remains, is there anything else you know?

**Comment by dudanogueira** [comment]
what about the  view-c-room permission?

**Comment by josep112** [comment]
Yes, I have placed an agent as an admin who has all permissions, but the error persists

**Comment by dudanogueira** [comment]
I had this exact same problem, and even I was admin, I was not able to access due to permissions. 

I was also assuming that as Admin I was able to do so. Check the permissions. If your user only has user and admin roles, go for each role and make sure they are according to the rule I wrote on the previous comments (**cannot have the permission view-joined-room** and also, it **must has the permission view-c-room**)
